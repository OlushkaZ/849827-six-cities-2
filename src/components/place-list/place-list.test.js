import React from 'react';
import renderer from 'react-test-renderer';
import PlaceList from './place-list.jsx';

it(`PlaceList correctly renders after relaunch`, () => {
  const tree = renderer
 .create(<PlaceList
   offers = {[{
     title: ``,
     coast: 0,
     isPremium: true,
     type: ``,
     src: ``,
   }]}
   // minutes={0}
   // onClick={jest.fn()}
 />)
 .toJSON();
  expect(tree).toMatchSnapshot();
});
