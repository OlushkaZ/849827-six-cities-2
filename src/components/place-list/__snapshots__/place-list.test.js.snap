// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaceList correctly renders after relaunch 1`] = `null`;
